.vscode/settings.json

```diff
@@ -18,5 +18,11 @@
     },
     "files.insertFinalNewline": true,
     "files.trimFinalNewlines": true,
-    "files.trimTrailingWhitespace": true
+    "files.trimTrailingWhitespace": true,
```

Comment:
I think that Tyler and I fixed this this morning if you merge master. LMK if that doesn't work for you.

Reply:
I tried merging master and got an 'Already up to date'... i'm not sure what to do about that

Reply:
This is taken care of now